src/test-reader/controllers/also-controller.ts

```diff
@@ -0,0 +1,41 @@
+import { EventEmitter } from "events";
+import { TestReaderEvents as ReadEvents } from "../../events";
+import type { Test } from "../../types";
+
+interface TreeBuilder {
+    addTrap: (trap: (test: Test) => void) => void;
+}
+
+export class AlsoController {
+    #eventBus: EventEmitter;
+
+    static create<T extends AlsoController>(this: new (eventBus: EventEmitter) => T, eventBus: EventEmitter): T {
+        return new this(eventBus);
+    }
+
+    constructor(eventBus: EventEmitter) {
+        this.#eventBus = eventBus;
+    }
+
+    in(matchers: string | RegExp | Array<string | RegExp>): this {
+        this.#addTrap(browserId => this.#match(browserId, matchers));
+
+        return this;
+    }
+
+    #addTrap(match: (browserId: string) => boolean): void {
+        this.#eventBus.emit(ReadEvents.NEW_BUILD_INSTRUCTION, ({ treeBuilder }: { treeBuilder: TreeBuilder }) => {
+            treeBuilder.addTrap(obj => {
+                if (match(obj.browserId)) {
+                    obj.enable();
+                }
+            });
+        });
+    }
+
+    #match(browserId: string, matchers: string | RegExp | Array<string | RegExp>): boolean {
+        return ([] as Array<string | RegExp>).concat(matchers).some(m => {
+            return m instanceof RegExp ? m.test(browserId) : m === browserId;
+        });
```

Comment:
Can we throw an error, if specified browser does not exist?
Why are you comparing strings as `===` instead of `browserId.includes(m)`? This way we could write `testplane.also.in("chrome-phone")` and run the test in all of the chrome phones (e.g. chrone-phone-121, chrome-phone-112, chrome-phone-dark...)

Reply:
> Can we throw an error, if specified browser does not exist?

We can do it, but it should be done in another PR in which we will also change logic in `testplane.skip` and `testplane.only`. To my mind they should work the same way.

> Why are you comparing strings as === instead of browserId.includes(m)?

For this case user should use regexp. Other controllers (`skip` and `only`) work in the same way. It means if we want match not strictly then it must be changed in all controllers.